client/src/auth/keycloakHelpers.ts

```diff
@@ -0,0 +1,52 @@
+import { KeycloakInitOptions } from "keycloak-js";
+
+// get tokens from localstorage
+export const getLocalTokens = () => {
+  const token = localStorage.getItem('token');
+  const refreshToken = localStorage.getItem('refreshToken');
+  return [token, refreshToken];
+};
+
+// save tokens to localstorage
+export const setLocalTokens = ({ token, refreshToken } : { token: string | undefined, refreshToken: string | undefined}) => {
+  if (token) localStorage.setItem('token', token);
+  if (refreshToken) localStorage.setItem('refreshToken', refreshToken);
+};
+
+export const getInitConfig = () : KeycloakInitOptions => {
+  const [token, refreshToken] = getLocalTokens();
+  if(token && refreshToken) return {
+    token,
+    refreshToken,
+    'promiseType': 'native',
+    'onLoad': 'login-required',
+  };
+  return {
+    'promiseType': 'native',
+    'onLoad': 'login-required',
+  }
+}
+
+// retrieve token and get authorization header
+export const getAuthHeader = () => {
```

Comment:
Also this seems to be antipattern. Keycloak will provide way to refresh token.
See: https://github.com/aerogear/aerogear-js-sdk/blob/master/packages/auth/src/Auth.ts#L121